blog/en/blog/2022/09/21/apache-apisix-v3-preview.md

```diff
@@ -0,0 +1,201 @@
+---
+title: "Apache APISIX 3.0: 10 Highlights of the Open Source API Gateway"
+authors:
+  - name: Ming Wen
+    title: Author
+    url: https://github.com/moonming
+    image_url: https://avatars.githubusercontent.com/u/26448043
+keywords:
+  - Ingress controller
+  - Open Source API Gateway
+  - API Management Platform
+  - The Best API Gateway
+  - Apache APISIX
+description: The open source API Gateway Apache APISIX beta version 3.0 is coming! We have selected ten essential features to give a brief introduction before the official release.
+tags: [Products]
+image: https://static-site.apiseven.com/wp-content/uploads/2022/09/APISIX.webp
+---
+
+> The open source API Gateway Apache APISIX beta version 3.0 is coming! We have selected ten essential features to give a brief introduction before the official release.
+
+<!--truncate-->
+
+[API Gateway](https://apisix.apache.org/docs/apisix/terminology/api-gateway/) has acted as an essential component for a long time. It has been committed to providing various functions such as [rate limiting](https://apisix.apache.org/docs/apisix/plugins/limit-req/), authentication (e.g., [Use Keycloak to secure APIs](https://apisix.apache.org/blog/2022/07/06/use-keycloak-with-api-gateway-to-secure-apis/)), and observability at the business level.
+
+With the iteration of server-side technologies, more and more services have begun to migrate from bare metal to Kubernetes, and the original monolithic architecture has gradually evolved into a microservice architecture. At the same time, enterprises have started to migrate their on-premise data centers to multi-cloud or hybrid clouds.
+
+With these technological advancements in the environment, the requirements for API gateways at the business level have become increasingly complex.
+
+## API Gateway Apache APISIX
+
+When [Apache APISIX](http://github.com/apache/apisix) was born, it hoped to help enterprises solve new problems in cloud-native environments and microservices. For example, it provides autoscaling of the business traffic through the fully dynamic feature and one-time modification to more conveniently achieve cluster management.
+
+Therefore, in the architectural design of APISIX, the data plane and the control plane are separated to achieve fully dynamic and cluster management, which is mainly accomplished by etcd components.
+
+![01.png](https://static.apiseven.com/2022/09/21/632ab5bd35b73.png)
```

Comment:
I think the picture descriptions should use a clearer description instead of just numbers

Reply:
Yeah, I agree. Double check all image descriptions please.